sensing/autoware_cuda_pointcloud_preprocessor/src/cuda_downsample_filter/cuda_voxel_grid_downsample_filter.cu

```diff
@@ -0,0 +1,471 @@
+// Copyright 2025 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware/cuda_downsample_filter/cuda_voxel_grid_downsample_filter.hpp"
+#include "autoware/cuda_downsample_filter/thrust_custom_allocator.hpp"
+
+#include <sensor_msgs/msg/detail/point_field__struct.hpp>
```

Comment:
Out of curiosity, why is the standard header not used here?

Reply:
This line was accidentally inserted by my editor. I creaned up the headre includes by 6b7383607